src/wasm.h

```diff
@@ -56,6 +56,13 @@
 
 namespace wasm {
 
+enum Features : uint32_t {
```

Comment:
Nit: Given we have `Features` and `FeatureSet`, would it make sense to call the enum of flags just `Feature`? Benefit would be that it would be clear that a variable named `features` is probably a `FeatureSet`

Reply:
Yeah, done.